src/utilities/renderer-helper.ts

```diff
@@ -179,7 +179,7 @@ class RendererHelper {
     async getProblem({
         sourceFilePath,
         problemSource,
-        problemSeed,
+        problemSeed = 666,
```

Comment:
Why is this defaulted to 666 here and defaulted to [here](https://github.com/rederly/backend/pull/134/files#diff-1fd04e20c23b5de82549b9541a8df404R878)?

Reply:
renderer will always default a missing problemSeed to 666 regardless

Reply:
I could remove it, probably makes sense, the doc had it labeled as required so I included it 😆

Reply:
I was just confused cause you changed the 666 above to null, so I was wondering why the inconsistency.